README.md

```diff
@@ -1,13 +1,17 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Learning goals
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+How to:
+* build an API in Node using Express
+* create routes in Express
+* do data manipulation in JavaScript - selecting, filtering and limiting arrays
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓